### PR TITLE
Fix more errors in next and mainline

### PR DESCRIPTION
**Pull request by nathanchance**
Each commit message should be self explanatory, let me know if there are any issues!

### CODE REVIEW

File: generator.yml [hunk 1]
```diff
@@ -962,7 +956,7 @@ builds:
   - {<< : *ppc64le_suse,      << : *next,             << : *clang,           boot: true,  llvm_version: *llvm_11}
   # https://github.com/ClangBuiltLinux/linux/issues/1023
   # https://github.com/ClangBuiltLinux/linux/issues/1143
```
Comment (nickdesaulniers) [inline generator.yml:958]: do these comments need to be deleted as well?

File: generator.yml [hunk 2]
```diff
@@ -1002,7 +996,7 @@ builds:
   - {<< : *ppc64le,           << : *stable-5_10,      << : *llvm,            boot: true,  llvm_version: *llvm_11}
   # https://github.com/ClangBuiltLinux/linux/issues/1023
   # https://github.com/ClangBuiltLinux/linux/issues/1143
```
Comment (nickdesaulniers) [inline generator.yml:998]: same

**Review by nickdesaulniers — APPROVED** [review]
thanks for the patches! 🍏 🌴

File: generator.yml (hunk 1 above)
Comment (nathanchance) [reply generator.yml:958]: Yes, good call!

File: generator.yml (hunk 2 above)
Comment (nathanchance) [reply generator.yml:998]: This one is fine because I did not touch 5.10.

**Comment by nathanchance** [comment]
Thanks for the review!

**Pull request merged**